pypit/artrace.py

```diff
@@ -999,6 +1006,9 @@ def trace_slits(slf, mstrace, det, pcadesc="", maskBadRows=False):
     else: # There's an order overlap
         rsub = edgbtwn[1]-(lval)
     """
+    if msgs._debug['trace']:
+        from xastropy.xutils import xdebug as xdb
```

Comment:
you're importing xdb, but then not using it  :-)

I think you either mean xdb.set_trace() on line 1011, otherwise you can simply delete line 1010